### PR TITLE
[WebConsole] Fix weird graph behavior when switching pipelines

### PR DESCRIPTION
**Pull request by Karakatiza666**
Fix unsubscribe from logs stream when leaving page

Fix https://github.com/feldera/feldera/issues/2745 : anomalous behavior of throughput graph

### CODE REVIEW

File: web-console/src/lib/functions/pipelines/changeStream.ts [hunk 1]
```diff
@@ -133,7 +133,6 @@ export const parseUTF8AsTextLines = (
   return {
     cancel: () => {
       reader.cancel()
-      stream.cancel()
```
Comment (gz) [inline web-console/src/lib/functions/pipelines/changeStream.ts:136]: this is unrelated?

**Review by gz — APPROVED** [review]

File: web-console/src/lib/functions/pipelines/changeStream.ts (hunk 1 above)
Comment (Karakatiza666) [reply web-console/src/lib/functions/pipelines/changeStream.ts:136]: Yes, this is another fix
